test/encoder.test.js

```diff
@@ -1,29 +1,25 @@
 const sample = require("./sample_tokens.json")
 const Encoder = require("../src/encoder").OpenPAYGOTokenEncoder
+const shared = require("../src/token").OpenPAYGOTokenShared
 
 describe("OpenPAYGOTokenEncoder test", () => {
   test("generateToken", () => {
     const encoder = new Encoder()
 
     sample.forEach((s) => {
       const data = s
-      try {
-        const { finalToken } = encoder.generateToken({
-          tokenType: data.token_type,
-          secretKeyHex: data.key,
-          count: data.token_count,
-          startingCode: data.starting_code,
-          restrictDigitSet: data.restricted_digit_set,
-          value: data.value_raw,
-          extendToken: false,
-        })
+      const useExtended = data.value_raw > shared.MAX_ACTIVATION_VALUE
```

Comment:
We should use this information from the newly added test cases. If the value is higher than allowed MAX_ACTIVATION_VALUE token generation should fail.

Reply:
Added a assertion on the test to ensure this.

Reply:
Added a assertion on the test to ensure this.